apps/emqx_exhook/src/emqx_exhook_server.erl

```diff
@@ -97,7 +97,7 @@ load(Name0, Opts0, ReqOpts) ->
                                io_lib:format("exhook.~s.", [Name])),
                     ensure_metrics(Prefix, HookSpecs),
                     %% Ensure hooks
-                    ensure_hooks(HookSpecs),
+                    ensure_hooks(HookSpecs, maps:get(priority, HooksOpts)),
```

Comment:
```suggestion
                    ensure_hooks(HookSpecs, maps:get(priority, HooksOpts, ?DEFAULT_HOOKED_PRIORITY)),
```

Reply:
The Default `priority` was got from `env/1`.
👍 It will change to macro style later.